lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/vbool/main.cpp

```diff
@@ -58,6 +63,38 @@ int main() {
   vBool.push_back(true);
   vBool.push_back(false);
   vBool.push_back(true);
+
+  // 48..=55
+  vBool.push_back(true);
+  vBool.push_back(true);
+  vBool.push_back(false);
+  vBool.push_back(true);
+  vBool.push_back(false);
+  vBool.push_back(true);
+  vBool.push_back(false);
+  vBool.push_back(true);
+
+  // 56..=63
```

Comment:
Any reason these new test-cases were added?

Reply:
Should've mentioned this. While I was testing, I made the mistake of calculating the byte index as `(idx / m_element_bit_size) * 8`. For reasons unknown to me, this made the tests pass (but it is wrong). I added the other elements to make sure that at least three elements of the underlying vector need to be accessed (to ensure that the calculation is actually correct).